TrafficCapture/dockerSolution/src/main/docker/migrationConsole/osiMigration.py

```diff
@@ -0,0 +1,385 @@
+#!/usr/bin/env python3
+
+from pathlib import Path
+from typing import List, Dict
+import argparse
+import logging
+import coloredlogs
+import boto3
+import os
+
+logger = logging.getLogger(__name__)
+DEFAULT_PIPELINE_NAME = "migration-assistant-pipeline"
+DEFAULT_PIPELINE_CONFIG_PATH = "./osiPipelineTemplate.yaml"
+AWS_SECRET_CONFIG_PLACEHOLDER = "<AWS_SECRET_CONFIG_PLACEHOLDER>"
+SOURCE_ENDPOINT_PLACEHOLDER = "<SOURCE_CLUSTER_ENDPOINT_PLACEHOLDER>"
+TARGET_ENDPOINT_PLACEHOLDER = "<TARGET_CLUSTER_ENDPOINT_PLACEHOLDER>"
+SOURCE_AUTH_OPTIONS_PLACEHOLDER = "<SOURCE_AUTH_OPTIONS_PLACEHOLDER>"
+TARGET_AUTH_OPTIONS_PLACEHOLDER = "<TARGET_AUTH_OPTIONS_PLACEHOLDER>"
+AWS_REGION_PLACEHOLDER = "<AWS_REGION_PLACEHOLDER>"
+STS_ROLE_ARN_PLACEHOLDER = "<STS_ROLE_ARN_PLACEHOLDER>"
+SOURCE_SECRET_ID_PLACEHOLDER = "<SOURCE_SECRET_ID_PLACEHOLDER>"
+SECRET_CONFIG_TEMPLATE = """
+pipeline_configurations:
+  aws:
+    secrets:
+      source-secret-config:
+        secret_id: <SOURCE_SECRET_ID_PLACEHOLDER>
+        region: <AWS_REGION_PLACEHOLDER>
+        sts_role_arn: <STS_ROLE_ARN_PLACEHOLDER>
+"""
```

Comment:
These three can be replaced done with standard string interpolation, because the variables are already filled above.  This would ensure that changing the variable name only has to happen in one place, or if we move away from these placeholders (to env vars or something else), this doesn't have to be changed. (same applies to lines 35-39)

```suggestion
SECRET_CONFIG_TEMPLATE = f"""
pipeline_configurations:
  aws:
    secrets:
      source-secret-config:
        secret_id: {SOURCE_SECRET_ID_PLACEHOLDER}
        region: {AWS_REGION_PLACEHOLDER}
        sts_role_arn: {STS_ROLE_ARN_PLACEHOLDER}
"""
```

Reply:
Ah, I slightly misunderstood how this was being used. I still think an f-string is a better way to do it, but will elaborate below